### ISSUE TITLE
Make Mapcreater

### ISSUE THREAD

**Issue by LiekeVanmulken** [GuusHamm/Call-of-Cactus#123]
Make a mapcreater so everyone can make a map and leave that to the community.

**Comment by teunw** [comment]
Should this be a standalone application, and does it need to be done before Oplevering II?

**Comment by GuusHamm** [comment]
That would be the best,not technically but it would be nice to have done on Iteration II

**Comment by teunw** [comment]
Added iteration, probably be standalone
